components/treeview/stories/treeview.test.js

```diff
@@ -77,6 +77,40 @@ export const TreeViewGroup = Variants({
 						},
 					],
 				},
+				{
+					id: "group3",
+					label: "Group 3 (Disabled items)",
+					link: "#",
+					icon: "Folder",
+					isDisabled: true,
+					isOpen: true,
+					items: [
+						{
+							id: "label5",
+							label: "Label 5",
+							link: "#",
+							icon: "Folder",
+							isDisabled: true,
+							items: [{
+								id: "label7",
+								label: "Label 7",
+								isDisabled: true,
+								link: "#",
+								icon: "Document",
+							}],
+						},
+						{
+							id: "label6",
+							label: "Label 6",
+							isDisabled: true,
+							link: "#",
+							icon: "Document",
+						}
+					],
+				},
+				{
+
+				},
```

Comment:
Was this empty object intentional?

Reply:
Thanks! It was not, just a remnant of when I thought I needed to add more disabled items then realized it wasn't necessary.